src/Storefront/Resources/app/storefront/src/scss/component/_address-manager.scss

```diff
@@ -0,0 +1,39 @@
+.address-manager-modal {
+  .btn-icon-hover-white {
+    &:hover, &:focus {
+      span.icon {
+        color: #FFFFFF;
```

Comment:
I am not the frontend pro, but don't we have predefined color variables, which we should use?